unstructured/partition/pdf.py

```diff
@@ -506,6 +507,33 @@ def _process_pdfminer_pages(
     return elements
 
 
+def _get_PDF_page_number(
+    filename: str = "",
+    file: Optional[bytes | IO[bytes]] = None,
+) -> int:
+    from pypdf import PdfReader
```

Comment:
I think we don't need local import in this function so this can got to  top imports.